### DISCUSSION TITLE
combining <TFHE rs>with<risc zero>

### DISCUSSION THREAD

**Discussion by GuangguangLu** [risc0/risc0#715] [Q&A] [answered]
I am combining <TFHE rs>with<risc zero>, but introducing dependencies tfhe={version="*", features=["boolean", "shortint", "integer", "x86_64 unix"]} in guest. toml  will result in an error.

Has anyone successfully combined these two tool libraries? I am very eager for your help.

error: failed to run custom build command for `concrete-csprng v0.3.0`
method_name: 
method_name: Caused by:
method_name:   process didn't exit successfully: `/root/my_project/target/riscv-guest/release/build/concrete-csprng-f9ee4ad05cba3593/build-script-build` (exit status: 101)

**Comment by pdg744** [comment, marked as answer]
There is discussion about making this work in #714 and an open issue for TFHE in #863
